pkg/tarutil/tar.go

```diff
@@ -260,23 +257,12 @@ func createFileInRoot(hdr *tar.Header, r io.Reader, rootDir string) error {
 // the file is omitted.
 type Filter func(hdr *tar.Header) bool
 
-// NoFilter does not filter or modify any files.
```

Comment:
This seems easy to test and the equivalent of /bin/true, why is it removed?

Reply:
No filtering is equivalent of nil or empty filter slice. I don't see how it can be useful?